test/libdnf/rpm/test_solv_query.cpp

```diff
@@ -302,8 +297,56 @@ void RpmSolvQueryTest::test_ifilter_requires() {
     {
         // Test QueryCmp::NEQ - string
         libdnf::rpm::SolvQuery query(sack.get());
-        std::vector<std::string> requires{"wget"};
+        std::vector<std::string> requires {"wget"};
         query.ifilter_requires(libdnf::sack::QueryCmp::NEQ, requires);
         CPPUNIT_ASSERT(query.size() == 288);
     }
 }
+
+void RpmSolvQueryTest::test_subject_solution() {
```

Comment:
Please rename the test. Tested function name is `resolve_pkg_spec` now.

Reply:
resplved